src/ontology/mondo-edit.obo

```diff
@@ -132346,50 +132319,73 @@ synonym: "optic nerve papillitis" EXACT [MESH:D010211]
 synonym: "optic papilla edema" EXACT [MESH:D010211]
 synonym: "optic papilla oedema" EXACT OMO:0003005 []
 synonym: "optic papillitis" EXACT [MESH:D010211]
+synonym: "papilledema" EXACT [MONDO:0002003]
 synonym: "papilledema associated with decreased intraocular pressure" EXACT [MESH:D010211]
 synonym: "papilledema associated with increased intracranial pressure" EXACT [MESH:D010211]
 synonym: "papillitis" EXACT [DOID:10175, MESH:D010211]
 synonym: "papillitis, optic" EXACT [MESH:D010211]
 synonym: "papillitis, optic nerve" EXACT [MESH:D010211]
+synonym: "papilloedema" EXACT [NCIT:C3307]
 synonym: "retinal edema" RELATED [MESH:D010211]
 synonym: "retinal Edemas" RELATED [MESH:D010211]
 synonym: "retinal oedema" RELATED OMO:0003005 []
 xref: DOID:10175 {source="EFO:1001074", source="MONDO:equivalentTo"}
+xref: DOID:146 {source="MONDO:equivalentTo"}
 xref: EFO:1001074 {source="MONDO:equivalentTo", source="MONDO:EFO"}
 xref: ICD10CM:H35.81 {source="DOID:10175"}
 xref: ICD10CM:H46.0 {source="EFO:1001074", source="DOID:10175"}
+xref: ICD10CM:H46.0 {source="DOID:146"}
 xref: ICD10CM:H47.1 {source="DOID:10175"}
+xref: ICD10CM:H47.1 {source="DOID:146"}
```

Comment:
These ICD10CM are duplicated. Sources should be grouped. I think a normalization step should take care of this.

Reply:
done